source/common/grpc/common.h

```diff
@@ -25,6 +25,34 @@ class Exception : public EnvoyException {
 
 class Common {
 public:
+  enum GrpcStatus {
```

Comment:
I'm happy to do so, but it would mean a pretty heavy protobuf include for status codes, and no way to indicate that a header is absent short of throwing an exception.    Think it's still worth it?